tests/integration/suite/actors/lock/reminders/self/grpc.go

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package reminders
+package self
```

Comment:
How about rename `self` to `local` is better, let us keep `self` away from package name.

Reply:
+1, and can you bring in all the changes I suggested from the other PR for this diff to the release branch?

Reply:
Done https://github.com/dapr/dapr/pull/8757/commits/73f697b70d56398eddfe5fe34c46ae1c119879e0